doc/ug/dv_methodology/index.md

```diff
@@ -43,23 +43,24 @@ We will explain some of the key items in those checklists in the remainder of th
 ## Documentation
 
 DV effort needs to be well documented to not only provide a detailed description of what tests are being planned, but also how the overall effort is strategized and implemented.
-The first is provided by the **testplan** document and the second, by the **DV plan** document.
+The first is provided by the **verification plan** document and the second, by the **DV plan** document.
 The [**project status**]({{< relref "doc/project/development_stages.md#indicating-stages-and-making-transitions" >}}) document tracks to progression of the effort through the stages.
 
 In addition to these documents, a nightly **regression dashboard** tabulating the test and coverage results will provide ability to track progress towards completion of the verification stages.
 
 To effectively document all these pieces, there are some key tooling components which are also discussed briefly below.
 
-### Testplan
+### Verification plan
 
-A testplan document captures at a high level, a list of tests that are being planned to verify all design features listed in the design specification.
-It is written in Hjson format and is made available in the corresponding `data` directory of each DUT.
+A verification plan consist of two parts a testplan that captures at a high level, a list of tests that are being planned to verify all design features listed in the design specification.
+A functional coverage plan that captures at high level a list of functional coverage points and coverage crosses needed to verify that the features listed in the design specification is tested by the list of tests.
+the verification plan is written in Hjson format and is made available in the corresponding `data` directory of each DUT.
 
 The Hjson schema enables this information to be human-writable and machine-parsable, which facilitates an automated and documentation-driven DV effort.
-The complete testplan is parsed into a data structure that serves the following purposes:
+The complete verification plan is parsed into a data structure that serves the following purposes:
 
-*  Provide the ability to insert the testplan as a table into the DV plan document itself, so that all of the required information is in one place
-*  Annotate the nightly regression results to allow us to track our progress towards executing the testplan
+*  Provide the ability to insert the testplan and coverage plan as a tables into the DV plan document itself, such that all of the required information is in one place
```

Comment:
Should be "as tables" (spare "a"). Also "so" was right: "such that" is also sort of correct, but sounds like we're trying a bit hard.

Reply:
ok I will fix that, to me "so" seem like the spoken language and "such" in writing.
but I am no expert. so I will go with your suggestion